internal/handlers/irc/handlers.go

```diff
@@ -92,15 +93,30 @@ func quitHandler(c Client) func(*girc.Client, girc.Event) {
 	}
 }
 
+/*
+kickHandler handles the event when a user is kicked from the IRC channel.
+*/
+func kickHandler(c Client) func(*girc.Client, girc.Event) {
+	return func(gc *girc.Client, e girc.Event) {
+		c.logger.LogDebug("kickHandler triggered")
+		if c.TelegramSettings != nil && c.TelegramSettings.ShowKickMessage {
```

Comment:
You can get rid of this check here
```go
if c.TelegramSettings != nil
```
It was unnecessarily added to the other handlers in a previous PR, I'm going to remove it from the other handlers in an upcoming PR

Reply:
Thanks for the review. I removed the check for `c.TelegramSettings != nil`.